src/olympia/users/models.py

```diff
@@ -177,9 +177,6 @@ def has_module_perms(self, app_label):
 
     backend = 'django.contrib.auth.backends.ModelBackend'
 
-    def is_anonymous(self):
```

Comment:
Ah, I knew we had our own implementation, I just didn't know django had it too :p